sdk/core/azure_core_test/src/proxy/matchers.rs

```diff
@@ -2,18 +2,33 @@
 // Licensed under the MIT License.
 
 use super::join;
-use azure_core::http::headers::{AsHeaders, HeaderName, HeaderValue};
+use azure_core::{
+    http::headers::{AsHeaders, HeaderName, HeaderValue},
+    Bytes,
+};
 use serde::Serialize;
 use std::{convert::Infallible, iter};
 
 // cspell:ignore headerless
 
+/// The headers ignored by default for [`CustomDefaultMatcherBody`];
+pub const DEFAULT_IGNORED_HEADERS: &[&str; 6] = &[
+    "date",
+    "request-id",
+    // cspell:disable-next-line
+    "traceparent",
+    "user-agent",
```

Comment:
User Agent is there because it means you can't use recordings across operating systems? I thought there was a user-agent specific sanitizer that removed the OS specific `()` section of the recordings.  FWIW, C++ doesn't filter user-agent and somehow recordings "just work".

Reply:
I copied this from .NET. Seems no other language has it, though. However, several sanitize it, but that seems problematic. Even if we sanitize the platform info, version info could change. I'd prefer to leave it unless you can think of any reason we would *want* to match against `user-agent`.